e2etest/bgptests/bgp.go

```diff
@@ -1136,6 +1136,43 @@ var _ = ginkgo.Describe("BGP", func() {
 					),
 				)
 			}
+
+			ginkgo.By("validating speakers running-config is updated after deleting the BFD profile")
```

Comment:
should we also validate if peers are referring to an existing BFD profile ?

Reply:
if the BFD profile doesn't exist we produce an error [here](https://github.com/metallb/metallb/blob/81a0de8f7a8e2d073590d77e55065f14df97d698/internal/config/config.go#L282) which is printed to the speakers log, i think checking the speakers log can be a bit tricky.

Reply:
oh! no, both states are just used to remove the entry from queue, looks this test is not feasible.

Reply:
we may use `metallb_k8s_client_update_errors_total` metric if that throws `SyncStateError`, but i'm not sure if this PR intended to do this.

Reply:
@sabinaaledort If you are interested in checking speakers logs you can use what I wrote on my PR
https://github.com/metallb/metallb/pull/1177/files#diff-dcd1cc413d712c08cb1ff01c2fb08dcaa495e4d2ed52e75d2367fc6b6da7ca82